tests/unit/test_my_annotator.py

```diff
@@ -0,0 +1,324 @@
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "sequence": "A",
+        "type": "LiteralSequenceExpression"
+    },
+    "type": "Allele"
+}
+
+deletion_output_normalized = {
+    "location": {
+        "end": 20003097,
+        "start": 20003096,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "length": 0,
+        "repeatSubunitLength": 1,
+        "sequence": "",
+        "type": "ReferenceLengthExpression"
+    },
+    "type": "Allele"
+}
+
+# https://www.ncbi.nlm.nih.gov/clinvar/variation/1687427/?new_evidence=true
+insertion_inputs = {
+    "hgvs": "NC_000013.11:g.20003010_20003011insG",
+    "spdi": ["NC_000013.11:20003010::G", "NC_000013.11:20003010:0:G"],
+    "gnomad": "13-20003010-A-AG"
+}
+
+insertion_output = {
+    "location": {
+        "end": 20003010,
+        "start": 20003010,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "sequence": "G",
+        "type": "LiteralSequenceExpression"
+    },
+    "type": "Allele"
+}
+
+gnomad_insertion_output = {
+    "location": {
+        "end": 20003010,
+        "start": 20003009,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "sequence": "AG",
+        "type": "LiteralSequenceExpression"
+    },
+    "type": "Allele"
+}
+
+# https://www.ncbi.nlm.nih.gov/clinvar/variation/1264314/?new_evidence=true
+duplication_inputs = {
+    "hgvs": "NC_000013.11:g.19993838_19993839dup",
+    "spdi": "NC_000013.11:19993837:GT:GTGT",
+    "gnomad": "13-19993838-GT-GTGT"
+}
+
+duplication_output = {
+    "location": {
+        "end": 19993839,
+        "start": 19993837,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "sequence": "GTGT",
+        "type": "LiteralSequenceExpression"
+    },
+    "type": "Allele"
+}
+
+duplication_output_normalized = {
+    "location": {
+        "end": 19993839,
+        "start": 19993837,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "length": 4,
+        "repeatSubunitLength": 2,
+        "sequence": "GTGT",
+        "type": "ReferenceLengthExpression"
+    },
+    "type": "Allele"
+}
+
```

Comment:
Add test for `normalized`
See here
https://github.com/ga4gh/vrs-python/blob/main/tests/extras/test_allele_translator.py#L200-L204

Reply:
Created